datasets/.gitignore

```diff
@@ -0,0 +1 @@
+*.csv
```

Comment:
Can you consolidate this `.gitignore` and the one in `subway_prediction_ml` into just the `.gitignore` at the root of the project?

Reply:
I did it this way for a reason, I don't remember the exact problem but this was a workaround for something.